src/lib/librtui/utils/utils.sh

```diff
@@ -80,6 +80,13 @@ __assert_t() {
 	fi
 }
 
+__external_script_type_check() {
+	__parameter_count_check 2 "$@"
+
+	__assert_f "$1"
+	bash -c "source /usr/lib/librtui/utils/utils.sh && source "$1" && __parameter_type_check "$2" function"
```

Comment:
```suggestion
	bash -c "source /usr/lib/librtui/utils/utils.sh && source '$1' && __parameter_type_check '$2' function"
```
你在双引号里面再用双引号就会结束字符串了啊，这样如果你变量里面有空格就会被分割成多个参数